packages/ensnode-utils/src/subname-helpers.spec.ts

```diff
@@ -75,3 +77,55 @@ describe("makeSubnodeNamehash", () => {
     );
   });
 });
+
+describe("labelByReverseAddress", () => {
+  const vitalikEthResolvedAddress = "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045";
```

Comment:
```suggestion
  const address = "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045";
```

Suggest to keep it simple and more generic.